src/ComScoreStreamingAnalytics.ts

```diff
@@ -39,7 +43,10 @@ export class ComScoreStreamingAnalytics {
 
     }
     this.registerPlayerEvents();
-
+    this.isMobile = !!navigator.userAgent && /Mobile/.test(navigator.userAgent);
+    if (this.isMobile) {
+      window.addEventListener('blur', this.blured);
```

Comment:
Are you sure that the `blur` event is sufficient to detect a home button press? What if there is a pop-up / tab-change etc.?